pyproject.toml

```diff
@@ -205,6 +202,7 @@ warn_unreachable = true
 strict_equality = true
 pretty = true
 plugins = ["pydantic.mypy"]
+files = ["eogrow"]
```

Comment:
what does this do exactly? what file is this?

Reply:
this informs mypy that by default it should check files in the `eogrow` folder, so you can just write `mypy` instead of `mypy eogrow`. I've grown lazy and don't feel like typing the entire command all the time :speak_no_evil:

Reply:
I thought it might be something like that, cool